backend/tilavarauspalvelu/api/graphql/types/application_section/serializers.py

```diff
@@ -289,6 +296,12 @@ def save(self, **kwargs: Any) -> CancellationOutput:
             EmailService.send_application_section_cancelled(application_section=self.instance)
             EmailService.send_staff_notification_application_section_cancelled(application_section=self.instance)
 
+            if has_access_codes:
+                # TODO: Only reschedule.
```

Comment:
TODO comment can be removed since unlike series deny this already calls reschedule (but add similar comment as below there)?

Reply:
Removed and added comment